.rultor.yml

```diff
@@ -1,5 +1,9 @@
 install:
-  - if ! pip3 -V ; then wget -O - https://bootstrap.pypa.io/get-pip.py | python3; fi
+  - >
+    if ! pip3 -V ; then
+    wget -O - https://bootstrap.pypa.io/get-pip.py |
```

Comment:
please indent `wget` by two spaces.

wget should save the file as `get-pip.py` and not use `|`.
Then on the next line `python3 get-pip.py`